packages/nestjs/package.json

```diff
@@ -50,8 +50,9 @@
     "@sentry/utils": "8.38.0"
   },
   "devDependencies": {
-    "@nestjs/common": "10.4.7",
-    "@nestjs/core": "10.4.7"
+    "@nestjs/common": "^8.0.0 || ^9.0.0 || ^10.0.0",
+    "@nestjs/core": "^8.0.0 || ^9.0.0 || ^10.0.0",
+    "@nestjs/event-emitter": "^2.0.0"
```

Comment:
Is there a reason we need this new dep?

Reply:
No actually we don't, I tried a different approach at first. I'll remove it